ktlint/src/test/kotlin/com/pinterest/ktlint/BaseCLITest.kt

```diff
@@ -101,15 +105,17 @@ abstract class BaseCLITest {
                     dir: Path,
                     attrs: BasicFileAttributes,
                 ): FileVisitResult {
-                    Files.createDirectories(dest.resolve(relativize(dir)))
+                    val relativeDir = dir.relativeToOrSelf(this@copyRecursively)
+                    (dest / relativeDir).createDirectories()
                     return FileVisitResult.CONTINUE
                 }
 
                 override fun visitFile(
                     file: Path,
                     attrs: BasicFileAttributes,
                 ): FileVisitResult {
-                    Files.copy(file, dest.resolve(relativize(file)))
+                    val relativeFile = file.relativeToOrSelf(this@copyRecursively)
+                    file.copyTo(dest / relativeFile)
```

Comment:
I would prefer to use the `Path.resolve` method instead of `/` operator to avoid confusion with the `/` as path separator.